src/static/libs/js/views/player-page/index.js

```diff
@@ -0,0 +1,119 @@
+require('../../navbar.js');
+require('../../auth.js');
+require('../../mediaPlayer.js');
+require('../../clipCreatedModal.js');
+require('../../createPlaylistModal.js');
```

Comment:
ideally all of that 'onload' logic would be in one location in your entry point bundle file and not spread out across all of those